svelte/src/routes/v2/+page.svelte

```diff
@@ -1,80 +1,84 @@
 <script lang="ts">
-    
-    import { onMount } from 'svelte';
-    import {cleanCanvas} from "./scripts/clean-canvas";
-    import {draw} from "./scripts/draw";
-    import {filter} from "./scripts/filter";
-    import {converter} from "./scripts/converter";
-	import {createGraphData} from "./scripts/create-graph-data";
+	import { onMount } from 'svelte';
+	import { cleanCanvas } from './scripts/clean-canvas';
+	import { draw } from './scripts/draw';
+	import { filter } from './scripts/filter';
+	import { converter } from './scripts/converter';
+	import { createGraphData } from './scripts/create-graph-data';
 	import RawDataInputer from './components/raw-data-inputer.svelte';
 	import ConfigChanger from './components/config-changer.svelte';
 	import DrawSettingsChanger from './components/draw-settings-changer.svelte';
 	import { onVertexCollapseClick } from './scripts/filter/collapse-vertices';
-	import type { ConfigInterface } from './types';
-	
+	import { extractAvailableEdgeType } from './scripts/helper';
+	import type { ConfigInterface, EdgeType } from './types';
+
 	let simulations: any[] = [];
-    
+
 	let rawData: any;
-    let convertedData: any;
-    const config:ConfigInterface = {
+	let convertedData: any;
+	let config: ConfigInterface = {
```

Comment:
Are we really going to change all these const to let?
(I know some people consider it wise to avoid const for non-primitive variables, do we want to adhere to that?)

EDIT: Did you put this through a linter or something? If so, we might want to agree on which linter to use, since there's a lot of small style changer throughout this document making PR's a bit hard to read.

Reply:
The main reason i changed it into let is because binding in svelte requires a non const variable. 

Yeah sorry about the linter, I'm using prettier currently. I think we can agree on a linter config